02 Writing Algorithms/28 Indicators/03 Manual Indicators/03 Create Manual Indicators.html

```diff
@@ -1,4 +1,4 @@
-<p>To create manual indicators, instantiate an indicator with its constructor. To view all of the available indicators and their constructors, see the Indicator Reference.</p>
+<p>To create manual indicators, instantiate an indicator with its constructor. To view all of the available indicators and their constructors, see the Supported Indicators.</p>
```

Comment:
- Add link to Supported Indicators page
- Remove "the" before "Supported Indicators".